### PR TITLE
Bench 195 update request models name ak java

### PR DESCRIPTION
**Pull request by MohammadIqbalAD**

### CODE REVIEW

File: src/main/java/com/answerdigital/answerking/controller/CategoryController.java [hunk 1]
```diff
@@ -50,7 +49,7 @@ public CategoryController(final CategoryService categoryService) {
                 content = { @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class)) })
     })
     @PostMapping
-    public ResponseEntity<CategoryResponse> addCategory(@Valid @RequestBody final AddCategoryRequest categoryRequest) {
+    public ResponseEntity<CategoryResponse> addCategory(@Valid @RequestBody final RequestModelsCategory categoryRequest) {
```
Comment (oleksiiRybak) [inline src/main/java/com/answerdigital/answerking/controller/CategoryController.java:52]: I think in API contract it is called Category.Request which we can name as CategoryRequest class in our codebase.

File: src/main/java/com/answerdigital/answerking/controller/OrderController.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ public OrderController(final OrderService orderService) {
     }
 
     @PostMapping
-    public ResponseEntity<Order> addOrder(@Valid @RequestBody final OrderRequest orderRequest) {
+    public ResponseEntity<Order> addOrder(@Valid @RequestBody final RequestModelsOrder orderRequest) {
```
Comment (oleksiiRybak) [inline src/main/java/com/answerdigital/answerking/controller/OrderController.java:38]: I think OrderRequest would fit better as a name,.

File: src/main/java/com/answerdigital/answerking/controller/CategoryController.java (hunk 1 above)
Comment (MohammadIqbalAD) [reply src/main/java/com/answerdigital/answerking/controller/CategoryController.java]: RequestModelsCategory changed to CategoryRequest.

File: src/main/java/com/answerdigital/answerking/controller/OrderController.java (hunk 2 above)
Comment (MohammadIqbalAD) [reply src/main/java/com/answerdigital/answerking/controller/OrderController.java]: RequestModelsOrder changed to OrderRequest.

**Review by AudriusPutys — APPROVED** [review]

**Review by oleksiiRybak — APPROVED** [review]

**Review by AndyFlintAnswerDigital — APPROVED** [review]

**Pull request merged**